modules/models/src/main/java/org/eclipse/xpanse/modules/models/policy/servicepolicy/ServicePolicyCreateRequest.java

```diff
@@ -26,6 +26,10 @@ public class ServicePolicyCreateRequest {
     @Schema(description = "The id of registered service template which the policy belongs to.")
     private UUID serviceTemplateId;
 
+
+    @Schema(description = "The flavor name which the policy belongs to.")
```

Comment:
please extend the description here to -

The flavor name which the policy belongs to. If a flavor is not provided, then the policy will be executed for during service deployment of all flavors.

Reply:
Updated.